core/domain/email_manager.py

```diff
@@ -2143,6 +2143,46 @@ def send_email_to_new_contribution_reviewer(
             feconf.NOREPLY_EMAIL_ADDRESS)
 
 
+def send_email_to_new_contribution_submit_questions(
+    recipient_id: str
+) -> None:
+    """Sends an email to user who is assigned to submit questions.
+
+    Args:
+        recipient_id: str. The ID of the user.
+    """
+
+    email_subject = 'You have been invited at Oppia to submit questions'
+
+    email_body_template = (
+        'Hi %s,<br><br>'
+        'This is to let you know that the Oppia team has granted you '
+        'the rights to submit questions.<br><br>'
+        'Link to the '
+        '<a href="https://www.oppia.org/contributor-dashboard">'
+        'Contributor Dashboard</a>.<br><br>'
+        'Thanks, and happy contributing!<br><br>'
+        'Best wishes,<br>'
+        'The Oppia Community')
+
+    if not feconf.CAN_SEND_EMAILS:
+        logging.error('This app cannot send emails to users.')
+        return
```

Comment:
Move this check to the beginning of the function.

Reply:
Done